apps/question-the-docs/frontend/src/components/DropdownMenu.jsx

```diff
@@ -0,0 +1,19 @@
+const DropdownMenu = ({ selectedOption, setSelectedOption }) => {
+
+  const handleOptionChange = (event) => {
+    setSelectedOption(event.target.value);
+  };
+
+  return (
```

Comment:
Yowza!  Is that how it works?!  You just return a block of HTML with template markings?  Amaze-ness!

I have to say that after a few months of this heavily typed code, all these function definitions seem a bit... naked now.